model-engine/model_engine_server/inference/batch_inference/requirements.txt

```diff
@@ -0,0 +1,6 @@
+ray==2.6.3
+git+https://github.com/vllm-project/vllm.git@4b61c6b669e368c6850531815940d9a542b9f223#egg=vllm
```

Comment:
curious: why not pin a published pypi version?

Reply:
ipv6 fix https://github.com/vllm-project/vllm/commit/4b61c6b669e368c6850531815940d9a542b9f223